chart/README.md

```diff
@@ -0,0 +1,82 @@
+
+    helm upgrade kuantifier oci://hub.opensciencegrid.org/iris-hep/kuantifier --version <latest-version> --values my-values.yaml
+
+## Configuration
+
+Helm charts are configured via a YAML [configuration file](https://helm.sh/docs/chart_template_guide/values_files/). The default
+configuration for Kuantifier is found at [values.yaml](./values.yaml). A custom values.yaml may be applied to a chart during
+installation or upgrade via the `--values` or `-f` helm command line option. The parameters of Kuantifier's values.yaml are documented
+below.
+
+### Values.yaml Parameters
+
+#### General Configuration
+* `.Values.outputFormat`: The outlet to report records to. Supports two values:
+  * `"ssmsend"`: Output to APEL via SSM.
+  * `"gratia"`: Output to [GRACC](https://gracc.opensciencegrid.org/) via 
+    [Gratia](https://github.com/opensciencegrid/gratia-probe/).
+* `.Values.cronjob`: Configuration for the Kubernetes [CronJob](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/)
+  object that periodically runs Kuantifier.
+  * `.schedule`: Cron expression for scheduling the CronJob
+  * `.priorityClassName`: [Priority Class](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/) for the
+    pods that run the Kuantifier job.
+  * `.ttlSecondsAfterFinished`: Duration to persist completed pods after completion. 
+* `.Values.pspName`: If specified, a [Pod Security Policy](https://kubernetes.io/docs/concepts/security/pod-security-policy/).
+    name to use. Deprecated in newer versions of Kubernetes.
+* `.Values.runAsNonRoot`: If true, run the pod as a [non-root user](https://kubernetes.io/docs/concepts/security/pod-security-standards/).
+
+* `.Values.dataVolumeSize`: The size of the volume where Kuantifier stores its intermediate results between querying Prometheus
+  and outputting to an external service. Note that Gratia output may require a significantly larger data volume size, as it operates
+  on non-summarized records.
+
+* `.Values.processor`: Configuration for the container that queries Prometheus to generate Job metric records.
+  * `.image_repository`: Docker image repository from which to pull the processor image.
+  * `.image_tag`: Optionally specify an image version.
+  * `.image_pull_policy`: [Image Pull Policy](https://kubernetes.io/docs/concepts/containers/images/#image-pull-policy) for the processor container.
+  * `.resources`: [Resource requests and limits](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for
+   the processor container. Allow approximately 10KB of memory per job record being processed.
+  * `.config`: Environment variables for the processor. See [KAPELConfig.py](../python/KAPELConfig.py) for full details. 
+  Must specify at least the following:
+    * `NAMESPACE`: The namespace in which pilot job containers are run. Kuantifier assumes every pod in the namespace is a pilot job.
```

Comment:
The code only cares about pods, "pilot job" is a bit specific to certain experiment workflows.

Reply:
I updated a bit more, in principle it does not even matter whether the pods are from jobs or potentially other controllers, but I have it in the back of my mind to document the requirement that pods need to be recorded on the cluster in a completed state. The fact that in practice this seems to work well for jobs but not other pod controllers has to do with specifics of Kubernetes' implementation of different controllers and/or how KSM works.